src/myr/reference.js

```diff
@@ -107,7 +107,7 @@ let transformations = [
   },
   {
     name: <span>setColor(<span className="string">color</span>)</span>,
-    description: <span>Changes the color of the cursor <a href='https://htmlcolorcodes.com/color-names/' target="_blank" rel="noopener noreferrer">using these color codes</a>, RGB, or HEX values. Default: 'red'</span>,
+    description: <span>Changes the color of the cursor <a href='https://htmlcolorcodes.com/color-names/' target="_blank" rel="noopener noreferrer">using these color codes</a>, or HEX values. Default: 'red'</span>,
```

Comment:
This shouldn't have been removed.

Reply:
that was an accident lol. i re-added it.